server/src/main/java/org/opensearch/index/codec/fuzzy/FuzzySetParameters.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index.codec.fuzzy;
+
+import java.util.function.Supplier;
+
+/**
+ * Wrapper for params to create a fuzzy set.
+ */
+public class FuzzySetParameters {
+    private final Supplier<Double> falsePositiveProbabilityProvider;
+    private final FuzzySet.SetType setType;
+
+    public static final double DEFAULT_FALSE_POSITIVE_PROBABILITY = 0.2047d;
```

Comment:
What's the rationale behind this false positivity rate?

Reply:
This is done based on the experiments here https://github.com/opensearch-project/OpenSearch/issues/4489#issuecomment-1724998489

I'm keeping the default with less memory footprint today while giving substantial throughput gains. This is a dynamic setting and customers can choose to set it on a live index based on their requirements.